changelog.md

```diff
@@ -1,3 +1,7 @@
+v0.4.10
+  fix concurrency issue when serializing dates. Also cache the value of "UTC" TimeZone locally to 
+  avoid contention to synchronized `getTimeZone` method.
```

Comment:
Please update description here regarding the thread locals too? Or alternatively we can explain this need with comments in the code